_episodes_rmd/06-tidy-data.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Tidying & packaging datasets"
+teaching: 30
```

Comment:
You think it will take 30 min to cover this? Seems a bit much for so little code. Maybe also include `spread`? And using the `gather(..., -variable_name)` minus ability to exclude a column from the gathering.

Reply:
WIP ;-) I didn't adjust the number after copying from a template.

But yes, `spread` and more about `gather` is a good idea!

Reply:
I didn't include more about this after all, because we can re-use so many other resources if people would like to learn about this. For the context of this inflammation dataset, I mostly want to get the column naming and tidying across.

Also, I would include `git commit`ing the steps in between or in the end, so I think 30min is realistic after all.